mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
That seems suspect ... it should not satisfy in that case, but at the same time I would not expect the SubmitQueue to need `repo` for anything -- that's access to private repositories as well as inviting people to be contributors. What endpoint was that?